src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -208,7 +206,13 @@ function BaseSelectionList<TItem extends ListItem>({
         }));
     };
 
-    const incrementPage = () => setCurrentPage((prev) => prev + 1);
+    const incrementPage = useCallback(() => {
+        // Calculate total options from sections
+        const totalOptions = sections.reduce((total, section) => total + (section.data?.length ?? 0), 0);
```

Comment:
Can't we use `flattenedSections.allOptions` here as it is already available and used earlier here https://github.com/Expensify/App/pull/69599/files#diff-750b08e6eebe347cd7ad8d106d3ea0192256cbd6217b8a97618239ee76b3404cL319

Reply:
ok, done @Pujan92